src/DrushStack.php

```diff
@@ -237,9 +237,11 @@ public function configDir($configDir)
         return $this;
     }
 
-    public function existingConfig()
+    public function existingConfig($existingConfig = false)
```

Comment:
I meant this `$existingConfig` parameter ;)
Since drush `--existing-config` is a flag with no options, there is no need for the Robo `existingConfig()` method to have a parameter...

Reply:
Ah, whoops :-)

Well to be honest, I added it to support chaining like we do in https://github.com/digipolisgent/robo-digipolis-drupal8/pull/54/files

I could remove it but then I'd have to write an if in that other PR :-)

Reply:
Ah, ok.
Then do you mind changing the default to `true`, so the method can be called without a parameter?
This would also suit your use case.

Reply:
Thanks & done.